packages/framer-motion/src/frameloop/index.ts

```diff
@@ -46,7 +45,7 @@ const processFrame = (timestamp: number) => {
     runNextFrame = false
 
     frameData.delta = useDefaultElapsed
-        ? defaultTimestep
+        ? 16.666
```

Comment:
Any reason for not making this a constant? I'd say that 1000/60 is clearer than this.

Reply:
(Don't feel strongly though, as it is kinda common knowledge)

Reply:
Works for me - done